boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -19,6 +19,25 @@
 import argparse
 import string
 
+BUNDLE_PERMITTED_RESOURCES = (
+    "ClusterServiceVersion"
```

Comment:
Erroneous space.

Reply:
Don't you need commas in here? Is this even compiling?